core/src/cats/sqlite_queries.inc

```diff
@@ -949,14 +949,46 @@ const char *BareosDbSqlite::query_definitions[] = {
 ,
 
 /* 0077_list_volumes_select_long_0 */
-"SELECT    MediaId,VolumeName,Slot,PoolId, "
-          "MediaType,FirstWritten,LastWritten,LabelDate,VolJobs, "
-          "VolFiles,VolBlocks,VolMounts,VolBytes,VolErrors,VolWrites, "
-          "VolCapacityBytes,VolStatus,Enabled,Recycle,VolRetention, "
-          "VolUseDuration,MaxVolJobs,MaxVolFiles,MaxVolBytes,InChanger, "
-          "EndFile,EndBlock,LabelType,StorageId,DeviceId, "
-          "LocationId,RecycleCount,InitialWrite,ScratchPoolId,RecyclePoolId, "
-          "Comment,Name AS Storage "
+"SELECT    Media.MediaId, "
+          "Media.VolumeName, "
+          "Media.Slot, "
+          "Media.PoolId, "
+          "(SELECT Name FROM Pool WHERE Pool.PoolId=Media.PoolId) AS Pool, "
```

Comment:
Can you really just change the query without changing the code that uses it?

Reply:
It adds extra output to the "llist media all" command. However, this should not hurt,